resource/doc/BuildForWindows.md

```diff
@@ -1,8 +1,39 @@
-# Prepare  [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/68i57b8ssasttngg?svg=true)](https://ci.appveyor.com/project/li-zhi/vmaf)
-  - Visual Studio 2015 on Windows
+# Building vmaf on Windows
+
+We are going to build vmaf using powershell. These steps are in accordance with the corresponding github action for building on Windows and have been tested successfully using a Windows10 machine.
+
+### Requirements
+Install these items and make sure they are available in path.
+
+- Python3.8
+- Doxygen (Download from their official [website](https://www.doxygen.nl/download.html))
+
+
+### Steps
+  - 1.Install [msys2](https://www.msys2.org/)
+  - 2.Install required msys2 packages using its shell:
   
-# Steps
-  - 1.open [vmaf.sln](../../vmaf.sln) in Visual Studio 2015 
-  - 2.Select Build => Batch Build from menu
-  - 3.Select the Solution Config as your want or Select All, then Build.
-  - 4.After Build, you will find all the build result in $(SolutionDir)/x64/$(Configuration), and there is examples.bat that you can run.
+        pacman -S mingw-w64-x86_64-ccache mingw-w64-x86_64-nasm mingw-w64-x86_64-pkg-config mingw-w64-x86_64-gcc mingw-w64-x86_64-meson mingw-w64-x86_64-ninja
+        
+  - 3.Open powershell in root directory of vmaf project and activate the python venv:
+        
+        .\.venv\Scripts\activate.ps1
+
+  - 4.Install required python packages. Note that the requirements are not needed to build the libvmaf. They are included for using the python scripts later if you want.
+
+        pip install meson
+        pip install -r python/requirements.txt
+        
+  - 5.Setup the required environment variables as follows:
+  
+         $Env:CC="ccache gcc"
+         $Env:CXX="ccache g++"
+         $Env:CFLAGS="-pipe -static -O3 -mtune=generic -D_FILE_OFFSET_BITS=64 -mthreads"
+         $Env:CXXFLAGS="-pipe -static -O3 -mtune=generic -D_FILE_OFFSET_BITS=64 -mthreads"
+         $Env:LDFLAGS="-pipe -static -static-libgcc -static-libstdc++"
+         $Env:Path = "C:\msys64\mingw64\bin;" + $Env:Path  
+
+  - 5.Setup the meson and build:
+
+        meson.exe setup libvmaf libvmaf/build --buildtype release --default-library static --prefix $((Get-Item .).FullName)\libvmaf\build\bin
```

Comment:
```suggestion
        meson setup libvmaf libvmaf/build --buildtype release --default-library static --prefix $PWD\libvmaf\build\bin
```

1. the provided `--prefix` is not one I would recommend since it will be under the libvmaf build folder, a more appropriate one might be `C:/msys2/mingw64` or `C:/vmaf-install` or something else
2. it is possible to call meson without the `.exe` suffix since windows will autoresolve `meson` to `meson.exe`
3. instead of pretending to `PATH`, you could just call meson with `C:\msys64\mingw64\bin\meson`